packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/add.html

```diff
@@ -1,24 +1,144 @@
 <oui-modal
-    ng-form="addServiceIntegration"
+    data-ng-form="addServiceIntegration"
     data-on-dismiss="$ctrl.cancel()"
     data-primary-label="{{:: 'pci_databases_service_integration_add_confirm' | translate}}"
-    data-primary-disabled="addServiceIntegration.$invalid"
+    data-primary-disabled="addServiceIntegration.$invalid || $ctrl.existingIntegrationError"
     data-primary-action="$ctrl.addServiceIntegration()"
     data-secondary-action="$ctrl.cancel()"
     data-secondary-label="{{:: 'pci_databases_service_integration_add_cancel' | translate}}"
     data-loading="$ctrl.processing"
-    data-heading="{{:: 'pci_databases_service_integration_add' | translate: {engineName: $ctrl.engineName} }}"
+    data-heading="{{:: 'pci_databases_service_integration_add' | translate}}"
 >
-    <oui-field
-        data-label="{{:: 'pci_databases_service_integration_add_service_label' | translate: {engineName: $ctrl.engineName} }}"
+    <oui-message data-type="warning" data-ng-if="$ctrl.showNoServiceMessage">
+        <div>
+            <span
+                data-translate="pci_databases_service_integration_add_error_no_available_service_1"
+            ></span>
+            <button
+                type="button"
+                class="oui-link"
+                data-ng-click="$ctrl.goToAddDatabase()"
+            >
+                <span
+                    data-translate="pci_databases_service_integration_add_error_no_available_service_link"
+                ></span>
+            </button>
+            <span
+                data-translate="pci_databases_service_integration_add_error_no_available_service_2"
+            ></span>
+        </div>
+        <ul>
+            <li
+                data-ng-repeat="engine in $ctrl.expectedEngines track by id"
+                data-ng-bind="engine"
+            ></li>
+        </ul>
+    </oui-message>
+
+    <oui-message
+        data-type="warning"
+        data-ng-if="$ctrl.existingIntegrationError"
     >
-        <oui-select
-            model="$ctrl.service"
-            name="service"
-            items="$ctrl.addableServicesList"
-            match="description"
-            required
+        <p
+            data-translate="pci_databases_service_integration_add_error_existing_integration"
+        ></p>
+    </oui-message>
+
+    <oui-message
+        data-type="warning"
+        data-ng-if="$ctrl.notCurrentServiceSelectedError"
+    >
+        <p
+            data-translate="pci_databases_service_integration_add_error_no_current_service"
+            data-translate-values="{serviceName: $ctrl.database.description}"
+        ></p>
+    </oui-message>
+
+    <div class="row">
+        <div class="col">
```

Comment:
**question:** Isn't this unnecessary?

Reply:
Absolutely, I removed it here: https://github.com/ovh/manager/pull/7186/commits/1deaf0646c832b5ecaf9fc76964d5ebb35a75b3a